### PR TITLE
HTML escape form data before showing

### PR DESCRIPTION
**Pull request by aaannz**
- fix for second part of #666

### CODE REVIEW

**Comment by coolo** [comment]
I'm confused that javascript shouldn't offer such a function already, but my google foo failed me (too)

File: t/ui/13-admin.t [hunk 1]
```diff
@@ -176,6 +176,45 @@ sub add_test_suite() {
...
+    $elem = $driver->find_element('.admintable tbody tr:last-child', 'css');
+    is($elem->get_text(), '=', "new row empty");
+    my ($name, $key, $value) = $driver->find_child_elements($elem, '//input[@type="text"]');
+    $name->send_keys($suiteName);
+    $key->send_keys($suiteKey);
+    $value->send_keys($suiteValue);
+    is($driver->find_element('//button[@title="Add"]')->click(), 1, 'added');
+    # leave the ajax some time
+    while (!$driver->execute_script("return jQuery.active == 0")) {
+        sleep 1;
```
Comment (okurz) [inline t/ui/13-admin.t]: can this be replaced by
```perl
t::ui::PhantomTest::wait_for_ajax();
```
as in d390fdcd0f6ef922a4b244e07e3d86ff9ca5fb4d ?

Comment (aaannz) [inline t/ui/13-admin.t]: Right, will do in new PR

Comment (okurz) [inline t/ui/13-admin.t]: no wait, already did in more places.

Comment (aaannz) [inline t/ui/13-admin.t]: Yeah, just noticed. You are too fast. :P

**Pull request merged**